client/v2/client.go

```diff
@@ -61,6 +70,9 @@ type Client interface {
 	// Query makes an InfluxDB Query on the database. This will fail if using
 	// the UDP client.
 	Query(q Query) (*Response, error)
+
+	// Close releases any resources a Client may be using.
+	Close() error
```

Comment:
I've always paired New/Close or Open/Close together depending on context. The same thing is kind of done in the [stdlib](https://golang.org/pkg/net/rpc/#NewClient). I would be against adding Open because it adds complexity to the API and isn't necessary for the http client.